### PR TITLE
[Nightly] adding IGAApplication

### PR DESCRIPTION
**Pull request by philbucher**
FYI @oberbichler @tteschemacher 

This app becomes more and more important (aslo for exacute) which is why it would be nice to get it tested in nightly build

### CODE REVIEW

**Comment by philbucher** [comment]
@roigcarlo it seems like skipping the CI works => i.e. if I know that I will push more changes then there is no need to fill CI with this :)

**Comment by philbucher** [comment]
obviously merging will still only be possible if CI passed

**Comment by philbucher** [comment]
for the record, we will also need a version of ANurbs in the testing account, same as what we did for MMG and EIgen

**Review by roigcarlo — DISMISSED** [review]
👍

**Comment by philbucher** [comment]
@loumalouomega in order to compile the IgaApplication a separate lib is required, same as for Eigen
How can I add it?

**Comment by philbucher** [comment]
ping @loumalouomega

**Comment by philbucher** [comment]
@loumalouomega ...?

**Comment by loumalouomega** [comment]
I haven't seen the notifications, sorry.

**Comment by loumalouomega** [comment]
I uploaded the files to a server (GDrive in my case), generate the public link and downloaded it in the script of the nightly build

**Comment by philbucher** [comment]
thanks @loumalouomega 

@roigcarlo I now added the automatic download of the ANurbs-lib (master-branch)

Can you please check? I think it is ok

File: scripts/build/nightly_build.sh [hunk 1]
```diff
@@ -32,13 +32,17 @@ unzip mmg.zip
...
+tar xzf AnurbsLibrary.tar.gz
+mv ${HOME}/ANurbs-master ${HOME}/ANurbs
 
 ## Step1: Prepare
 wget http://www.logix.cz/michal/devel/smtp-cli/smtp-cli
 chmod 777 smtp-cli
 
-wget https://github.com/KratosMultiphysics/Kratos/archive/master.tar.gz
-tar xzf master.tar.gz
+wget https://github.com/KratosMultiphysics/Kratos/archive/master.tar.gz -O KratosMaster.tar.gz
```
Comment (philbucher) [inline scripts/build/nightly_build.sh:44]: I am doing this to prevent name-clashes => default name is just "master.tar.gz"

**Review by roigcarlo — APPROVED** [review]
No problems from my side, Just tested it and seems to work fine :+1:

**Comment by philbucher** [comment]
thanks

**Pull request merged**